blockchain/indexers/existsaddrindex.go

```diff
@@ -453,47 +445,41 @@ func (idx *ExistsAddrIndex) disconnectBlock(dbTx database.Tx, block, parent *dcr
 func (idx *ExistsAddrIndex) addUnconfirmedTx(tx *wire.MsgTx, isTreasuryEnabled bool) {
 	isSStx := stake.IsSStx(tx)
 	for _, txIn := range tx.TxIn {
-		if txscript.IsMultisigSigScript(txIn.SignatureScript) {
-			// Note that the functions used here require v0 scripts.  Hence it
-			// is used for the script version.  This will ultimately need to
-			// updated to support new script versions.
-			const scriptVersion = 0
-			rs := txscript.MultisigRedeemScriptFromScriptSig(
-				txIn.SignatureScript)
-			class, addrs, _, err := txscript.ExtractPkScriptAddrs(
-				scriptVersion, rs, idx.chain.ChainParams(),
-				isTreasuryEnabled)
+		// Note that the functions used here require v0 scripts.  Hence it
+		// is used for the script version.  This will ultimately need to
+		// updated to support new script versions.
```

Comment:
s / ... need to updated to ... / ... need to be updated to ...

Reply:
Added.  Also updated another case with the same typo.